common/txmgr/types/fee_estimator.go

```diff
@@ -14,9 +14,13 @@ const (
 	OptForceRefetch Opt = iota
 )
 
+type Fee interface {
```

Comment:
Add comments to explain this interface.

Reply:
this interface is simplified down to `fmt.Stringer`